### ISSUE TITLE
Performance issue. Calling removeAllEnpoints method does not delete some endpoints data from DOM. 

### ISSUE THREAD

**Issue by lexeek** [jsplumb/jsplumb#1110]
In case jsPlumb config has `endpointHoverStyle` options configuration then method removeAllEndpoints does not remove all endpoint elements from DOM.

```
   jsPlumbInstance = newInstance({
        container: container.value,
        endpointHoverStyle: {
          fill: "#808080",
          stroke: "#808080",
          strokeWidth: 6,
        },
      });

```

I created an example [Jsplumb endpoints with hover style option](https://codesandbox.io/s/frosty-morning-r0zos4?file=/src/App.vue:1026-1044)

In this example, you can click on the button inside the node for deleting the node endpoints by calling the method jsPlumbInstance.removeAllEndpoints. The result is that endpoints are deleted but you can find that some elements with hover styles still exist in DOM.

1. Nodes with andpoints and connections.
![image](https://user-images.githubusercontent.com/1452043/154297811-e1e697c3-76eb-4032-98d9-9e7d70ec962b.png)

2. Connections and endpoints are removed but some elements still  exist in DOM
![image](https://user-images.githubusercontent.com/1452043/154298284-b94256c4-614d-4e0c-9d98-ca606110f578.png)

**Comment by sporritt** [comment]
it's pretty much impossible to debug that example. i dont suppose you could reproduce without all the vue stuff could you.

**Comment by sporritt** [comment]
..actually dont worry, i can reproduce this in a unit test

**Comment by lexeek** [comment]
@sporritt Is this issue fixed?

**Comment by sporritt** [comment]
yes it is fixed in 5.5.0, which was released yesterday.
